closure/repositories.bzl

```diff
@@ -675,26 +675,23 @@ def com_google_jsinterop_annotations():
     )
 
 def com_google_protobuf():
-    # Note: Protobuf 3.6.0+ is going to use C++11
     http_archive(
         name = "com_google_protobuf",
-        strip_prefix = "protobuf-3.5.1",
-        sha256 = "826425182ee43990731217b917c5c3ea7190cfda141af4869e6d4ad9085a740f",
+        strip_prefix = "protobuf-3.6.1.2",
+        sha256 = "2244b0308846bb22b4ff0bcc675e99290ff9f1115553ae9671eba1030af31bc0",
         urls = [
-            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.5.1.tar.gz",
-            "https://github.com/google/protobuf/archive/v3.5.1.tar.gz",
+            "https://github.com/protocolbuffers/protobuf/archive/v3.6.1.2.tar.gz",
         ],
     )
 
 def com_google_protobuf_js():
     http_archive(
         name = "com_google_protobuf_js",
         urls = [
-            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.5.1.tar.gz",
-            "https://github.com/google/protobuf/archive/v3.5.1.tar.gz",
+            "https://github.com/google/protobuf/archive/7b28271a61a3da0a37f6fda399b0c4c86464e5b3.tar.gz",
```

Comment:
Currently all of the external deps here are mirrored from mirror.bazel.build. I'm not sure if that's a policy/best practice or not but your change doesn't follow that.

Reply:
Can you (or someone else) upload https://github.com/protocolbuffers/protobuf/archive/v3.6.1.2.tar.gz" to https://mirror.bazel.build/github.com/google/protobuf/archive/v3.6.1.2.tar.gz? Then I can re-add new mirror URL.

Reply:
I only have write access to rules_closure and not even sure if this is important.
Somebody from Bazel team should clarify if bazel mirrors has any value-add here.
@dslomov @laurentlb

Reply:
`should clarify if bazel mirrors has any value-add here`
I have no idea about the mirrors, but most other rules don't use them.

Reply:
TensorFlow uses bazel mirrors a lot, because it's more stable than the github link.